go/src/koding/kites/kloud/stack/whoami.go

```diff
@@ -16,6 +16,16 @@ type Team struct {
 	Subscription string `json:"subscription,omitempty"`
 }
 
+// IsSubActive checks if Team's subscription is active.
+func (t *Team) IsSubActive() bool {
```

Comment:
lets not duplicate the logic, and have the critical logic in one place, please refactor this and the one we have in group.go to use the exact same code.